src/main/java/org/jenkinsci/plugins/gitclient/ChangelogCommand.java

```diff
@@ -84,6 +84,18 @@
      */
     ChangelogCommand includes(ObjectId rev);
 
+    /**
+     * Sets the firstParent private variable to true
+     */
+    ChangelogCommand firstParent();
+
+    /**
+     * Sets the firstParent private variable to boolean param
+     *
+     * @param firstParent a {@link java.lang.boolean} object.
```

Comment:
Please don't include the type in description again (even if there are some existing Javadocs like this} - `java.lang.boolean` is not even valid.